src/main/java/oz/yamyam_map/module/restaurant/entity/ReviewRating.java

```diff
@@ -0,0 +1,36 @@
+package oz.yamyam_map.module.restaurant.entity;
+
+import jakarta.persistence.Embeddable;
+
+@Embeddable
+public class ReviewRating {
+
+	private Long totalReviews;
+	private Long totalScore;
+	private double averageScore;
+
+	// JPA를 적용하기 위해 기본 생성자 추가
+	protected ReviewRating() {
+		this.totalReviews = 0L;
+		this.totalScore = 0L;
+		this.averageScore = 0.0;
+	}
+
+	public ReviewRating(Long totalReviews, Long totalScore) {
+		this.totalReviews = totalReviews;
+		this.totalScore = totalScore;
+		this.averageScore = totalReviews > 0 ? (double)totalScore / totalReviews : 0.0;
+	}
+
+	public void addReview(int newScore) {
+		totalReviews++;
+		totalScore += newScore;
+		calculateNewReviewRating();
+	}
+
+	private void calculateNewReviewRating() {
+		averageScore = (double)totalScore / totalReviews;
+	}
```

Comment:
💭 레스토랑 클래스에서 리뷰 관련 내용을 캡슐화 한다는 점에서 `ReviewRating` 도입하는 거 너무 좋습니당!👍🏻 그런데 vo에서 값을 변경하는게 아주 쪼금 어색한 것 같아요! 아래처럼 새로운 객체를 만들어서 리턴하면, 불변성을 지킬 수 있을 것 같습니다!! 그런데 이건 FM적인 내용이니.. 기존대로 가셔도 좋아용☺️
```suggestion
	public ReviewRating addReview(int newScore) {
		return new ReviewRating(totalReviews + 1, totalScore + newScore);
	}
```

Reply:
오 저는 엔티티의 일부라고 생각해서 이 생각을 못했었는데 그렇겠네요 
감사해요 👍 수정하겠습니다!